web/pdf_viewer.component.js

```diff
@@ -25,31 +26,28 @@ import { DownloadManager } from './download_manager.js';
 import { GenericL10n } from './genericl10n.js';
 import { PDFFindController } from './pdf_find_controller.js';
 import { PDFHistory } from './pdf_history.js';
-import pdfjsLib from './pdfjs.js';
 import { PDFPageView } from './pdf_page_view.js';
 import { PDFSinglePageViewer } from './pdf_single_page_viewer';
 import { PDFViewer } from './pdf_viewer.js';
 
-let { PDFJS, } = pdfjsLib;
-
-PDFJS.PDFViewer = PDFViewer;
-PDFJS.PDFSinglePageViewer = PDFSinglePageViewer;
-PDFJS.PDFPageView = PDFPageView;
-PDFJS.PDFLinkService = PDFLinkService;
-PDFJS.SimpleLinkService = SimpleLinkService;
-PDFJS.TextLayerBuilder = TextLayerBuilder;
-PDFJS.DefaultTextLayerFactory = DefaultTextLayerFactory;
-PDFJS.AnnotationLayerBuilder = AnnotationLayerBuilder;
-PDFJS.DefaultAnnotationLayerFactory = DefaultAnnotationLayerFactory;
-PDFJS.PDFHistory = PDFHistory;
-PDFJS.PDFFindController = PDFFindController;
-PDFJS.EventBus = EventBus;
-
-PDFJS.DownloadManager = DownloadManager;
-PDFJS.ProgressBar = ProgressBar;
-PDFJS.GenericL10n = GenericL10n;
-PDFJS.NullL10n = NullL10n;
+const pdfjsVersion = PDFJSDev.eval('BUNDLE_VERSION');
```

Comment:
I'm not entirely sure why these two constants are added here because they weren't in the code before and they are not exported. What am I missing here?

Reply:
This was probably not entirely relevant to the rest of the patch, however it addresses an in my opinion existing shortcoming.
Note how both https://github.com/mozilla/pdf.js/blob/master/src/pdf.js and https://github.com/mozilla/pdf.js/blob/master/src/pdf.worker.js contains this information (without exporting it). That can be very handy if you're looking at the built files and want to know which version/build it corresponds to.